### PR TITLE
Pins BrowseEverything and BigDecimal to current version.

### LINKED ISSUE #931: BrowseEverything not displaying providers
### Descriptive summary

When a user clicks on add a cloud file the screen fades to grey and but they do not see the module that allows them to login and select their cloud provider.

### Expected behavior

It is expected that the user will see the module and providers.

### Steps to reproduce the behavior

1. Do this
1. Then do this...

### Related work

Link to related issues or prior related work here.

### PR DESCRIPTION
**Pull request by scherztc**
Fixes #931 

Present short summary (50 characters or less)

The BE model was greyed out and the user was unable to select and providers.  Once we connected up the providers, the user was able to select a file , but was unable to add it to the work.  It would spin at 100%.

We decided to pin the version of 
BE to 1.0.2 
BigDecimal to 1.3.5

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Application
+≈# Application
```
Comment (crowesn) [inline .env.development:1]: What is the purpose of this equal sign? Typo?

**Review by crowesn — APPROVED** [review]
This passes manual testing on qa - I noted that the file_system option is active and I don't remember that being available on production. Thumbs up pending ci passing and my comment on the dev env file.

File: .env.development (hunk 1 above)
Comment (scherztc) [reply .env.development:1]: Yeah... Typo.

Comment (scherztc) [reply .env.development]: Fixed and pushed back up.

**Comment by crowesn** [comment]
@scherztc Is google drive broken? I see that's been eliminated as a provider.

**Pull request merged**